submission/submission-task/src/main/kotlin/ac/uk/ebi/biostd/submission/task/SubmissionApp.kt

```diff
@@ -0,0 +1,64 @@
+package ac.uk.ebi.biostd.submission.task
```

Comment:
Since this is a new project, I think we should use the right package domain naming since the beginning: `uk.ac.ebi`